cassandra/deserializers.pyx

```diff
@@ -127,8 +127,8 @@ cdef class DesCounterColumnType(DesLongType):
 
 cdef class DesDateType(Deserializer):
     cdef deserialize(self, Buffer *buf, int protocol_version):
-        cdef double timestamp = unpack_num[int64_t](buf) / 1000.0
-        return datetime_from_timestamp(timestamp)
+        cdef int64_t timestamp_in_us = unpack_num[int64_t](buf) * 1000
```

Comment:
well, I normally try to avoid scaling overflows if I can. I haven't done the math in this case to see if any overflowing ``us`` count would even be representable in ``datetime``.
Regardless, since we're using signed types I think we should change it to avoid surprising 'wrapped' dates for large counts.